source/reference/api/reverse.txt

```diff
@@ -40,8 +40,15 @@ To use the ``reverse`` endpoint:
 - You must have started your original sync operation with the
   ``reversible`` and ``enableUserWriteBlocking`` options set to
   ``true``.
-
 - ``mongosync`` must be in the ``COMMITTED`` state.
+- :ref:`Unique indexes <index-type-unique>` must be formatted properly.
+  Format indexes with the :dbcommand:`collMod` command. See `this example
+  <https://www.mongodb.com/docs/v6.0/reference/command/collMod/#convert-an-existing-index-to-a-unique-index>`__
+  for more details.
```

Comment:
I don't think this is relevant here. On reverse, mongosync converts the unique indexes to non-unique and that's why the unique indexes format matters here.

I think it would be more straightforward to say: "To use reverse replication, [unique indexes](https://www.mongodb.com/docs/master/core/index-unique/?&_ga=2.162030843.612489097.1658153614-1269726939.1563474234#std-label-index-type-unique) on the original source cluster must be formatted properly in the newer internal format. Source and destination clusters must also be MongoDB 6.0 or later. If an upgraded cluster has unique indexes that were created in MongoDB 4.2 or earlier, you must [resync](https://www.mongodb.com/docs/v6.0/tutorial/resync-replica-set-member/) all nodes in your cluster"